src/features/editor/components/input/ToolDropdownMenu.tsx

```
@@ -0,0 +1,68 @@
+
+interface ToolDropdownMenuProps {
+  onSelect: (toolName: string) => void;
+  onClose: () => void;
+  className?: string; // For positioning
+  style?: React.CSSProperties; // For dynamic positioning (Portal)
+  onMouseEnter?: () => void;
+  focusedIndex?: number | null; // 키보드 네비게이션용 포커스 인덱스
+  onFocusChange?: (index: number) => void; // 마우스 호버 시 포커스 변경 업데이트
+}
+
+export const TOOLS = [
```

ESLint

[error] Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components.

(react-refresh/only-export-components)